test/api.js

```diff
@@ -2768,7 +2768,7 @@ describe('/v1', function() {
       });
     });
 
-    it('should not return the email if email:false for profile:email scope', function() {
+    it('fails for payload having email:false', function() {
```

Comment:
Because we have to allow `email: false` for one train until all services stop sending it, these test updates need to wait one train.